### ISSUE TITLE
The "composer/installers" plugin requires composer-plugin-api 1.0.0

### ISSUE THREAD

**Issue by ollietreend** [roots/bedrock#236]
Hey,

I'm sure it's not a bug with bedrock (after all, I've seen [similar](https://travis-ci.org/silverstripe/silverstripe-framework/jobs/105063132) [notices](https://travis-ci.org/WP-API/WP-API/jobs/104228640) [elsewhere](https://travis-ci.org/hechoendrupal/DrupalConsole/jobs/104821358)) but I'm hoping to glean some insight into its cause and consequences.

This is the output I get when installing bedrock with composer:

```
Installing roots/bedrock (1.5.1)
  - Installing roots/bedrock (1.5.1)
    Downloading: 100%

Created project in /var/www
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
  - Installing composer/installers (v1.0.22)
    Downloading: 100%

The "composer/installers" plugin requires composer-plugin-api 1.0.0, this *WILL* break in the future and it should be fixed ASAP (require ^1.0 for example).
  - Installing johnpbloch/wordpress-core-installer (0.2.1)
    Downloading: 100%

  - Installing johnpbloch/wordpress (4.4.1)
    Downloading: 100%

  - Installing oscarotero/env (v1.0.1)
    Downloading: 100%

  - Installing vlucas/phpdotenv (v2.2.0)
    Downloading: 100%

Generating autoload files
```

I can't help but notice the warning message from `composer/installers` about it requiring `composer-plugin-api 1.0.0`.

Is this any action required for bedrock? Or is this something that will be resolved upstream?

**Comment by swalkinshaw** [comment]
Thanks for pointing this out. Fixed in 9b5ffe83efcad892fabdc9dc8f763fa11cf7725c

**Comment by swalkinshaw** [comment]
Should note that you can fix this locally by just running `composer update`.
